gubernator/view_pr.py

```diff
@@ -96,6 +97,20 @@ def get(self, path, pr):
             max_builds=max_builds, header=headings, repo=repo, rows=rows, path=path))
 
 
+def get_acks(login, prs):
+    acks = {}
+    result = ghm.GHUserState.make_key(login).get()
+    if result:
+        acks = result.acks
+        if prs:
+            stale = set(acks) - set(pr.key.id() for pr in prs)
```

Comment:
It is not clear to me what prs means... prs in what kind of state? I guess PRs still needing my attention?